beetsplug/subsonic.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Subsonic plugin for Beets
+# Allows to trigger a library scan on Subsonic
+# after music has been imported with Beets
+# Your Beets configuration file should contain
+# a subsonic section like the following:
+#   subsonic:
+#       host: 192.168.x.y (the IP address where Subsonic listens on)
+#       port: 4040 (default)
+#       user: <your username>
+#       pass: <your password>
+# Plugin wrote by maffo999
+
+"""Updates Subsonic library on Beets import
+"""
+from __future__ import division, absolute_import, print_function
+
+from beets.plugins import BeetsPlugin
+import requests
+import string
+import hashlib
+import random
+
+
+class Subsonic(BeetsPlugin):
+    def __init__(self):
+        super(Subsonic, self).__init__()
+        self.register_listener('import', self.loaded)
+
+    def loaded(self):
+        host = self.config['host'].get()
+        port = self.config['port'].get()
```

Comment:
Consider using `self.config.add()` to provide defaults for these options. For example, `host` might default to `127.0.0.1`, and it looks like there's a reasonable default port too. See other plugins for examples of how to provide defaults.

Reply:
I avoided that as people could have (like me) the two software running on different machines.
While I agree that a default configuration is better than none at all, it still poses the risk of having a non-working plugin.

Reply:
Hmm... I’m not quite sure I follow. Why would having multiple instances be incompatible with having a default? I imagine most people will want to customize most things, but perhaps not all the options all of the time.